roles/horizon/tasks/logging.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: set up log rotation for horizon
+  logrotate: name=horizon path=/var/log/apache2/*.log
+  args:
+    options:
+      - daily
+      - missingok
+      - rotate 7
+      - compress
```

Comment:
overwritten config is this 
```
/var/log/apache2/*.log
{
  daily
  missingok
  rotate 7
  compress
}
```

which matches other services. i'll push my changes up for review if this is sufficient.